### ISSUE TITLE
Support react 18 and the used version : react-select@4.3.1

### ISSUE THREAD

**Issue by benramsay** [Tintef/react-google-places-autocomplete#289]
Since I upgraded to React 18, I started getting warnings. 
Could you please upgrade the **react-select version** to the latest please. 

This is an example of the warnings I have :

```
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.8.0 || ^17.0.0" from react-select@4.3.1
npm WARN node_modules/react-google-places-autocomplete/node_modules/react-select
npm WARN   react-select@"^4.3.1" from react-google-places-autocomplete@3.4.0
npm WARN   node_modules/react-google-places-autocomplete
```

**Comment by C0retek** [comment]
Is this planned to be update to React 18 anytime soon?

**Comment by Tintef** [comment]
As soon as I have some free time 😞

**Comment by Tintef** [comment]
This is published now under v4.0.0!
